### ISSUE TITLE
0.4 for train

### ISSUE THREAD

**Issue by Shaiken** [uihilab/HydroLSTM#1]
data_valid, data_train = train_test_split(data_train_valid, test_size=0.4, shuffle= False)

if your csv 
data (valid) (46033, 288)
data train (18413, 288)
data valid (27620, 288)
data test (8017, 288)
